doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -577,8 +577,11 @@ The current implementation recognizes the following built-in quotations:
 - ``ident``, which parses identifiers (type ``Init.ident``).
 - ``constr``, which parses Rocq terms and produces an-evar free term at runtime
   (type ``Init.constr``).
+- ``lconstr``, which is the same as ``constr`` except at a much higher level.
```

Comment:
The documentation should probably be explicit about the level, i.e. this should read "which is equivalent to constr but at precedence level 200" and similarly for the one below.

Reply:
Adopted:)